pom.xml

```diff
@@ -41,8 +41,8 @@
     </developers>
 
     <scm>
```

Comment:
I think the git commit will fail during the mvn scm:checkin command. I'd update to match the [habushu pom.xml](https://github.com/TechnologyBrewery/habushu/blob/dev/pom.xml) - `https` and remove the `git@`.
```
<scm>
        <connection>scm:git:https://github.com/TechnologyBrewery/habushu.git</connection>
        <developerConnection>scm:git:https://github.com/TechnologyBrewery/habushu.git</developerConnection>
        <url>https://github.com/TechnologyBrewery/habushu/</url>
        <tag>HEAD</tag>
    </scm>
```

Reply:
Fixed.